### DISCUSSION TITLE
Refresh token and Access token

### DISCUSSION THREAD

**Discussion by dmdgpdi** [Kernel360/F2-BAGUNI#298] [메모장]
[인증과 인가를 안전하게 처리하기 (Refresh Token Rotation)](https://velog.io/@chchaeun/%EC%9D%B8%EC%A6%9D%EA%B3%BC)
[Access Token & Refresh Token 원리](https://inpa.tistory.com/entry/WEB-%F0%9F%93%9A-Access-Token-Refresh-Token-%EC%9B%90%EB%A6%AC-feat-JWT)
[[JWT] Access Token의 한계와 Refresh Token의 필요성](https://colabear754.tistory.com/179)
[JWT Refresh Token 사용하기](https://yookeun.github.io/java/2023/12/30/jwt-refresh/)

링크에서 탈취 시나리오, Refresh token으로 재발급 등의 내용이 담겨있어서 공유드립니다.

**Comment by kimminkyeu** [comment]
감사합니다!
